### PR TITLE
upgrade to backbone.wreqr v1.3.2

### PR DESCRIPTION
**Pull request by gbicou**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>backbone-wreqr</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>1.3.2</version>
```
Comment (jamesward) [inline pom.xml:14]: This needs to be `1.3.2-SNAPSHOT` since Maven does the actual release.

**Comment by gbicou** [comment]
sorry, reopening a pull request

**Comment by jamesward** [comment]
In the future you can just update your branch to update an existing PR.

**Comment by gbicou** [comment]
Ok.

**Pull request closed without merging**